test/python/gemm/frost/test_matmul_epilogue_fusion.py

```
@@ -2561,10 +2563,17 @@ def test_smem_d_reserve_matches_what_the_templates_stage(cfg_name: str, cta_grou
     from cudnn.gemm.frost.dtypes import DTYPE_BYTES
     from cudnn.gemm.frost.tile_config import by_name
 
+    from cudnn.gemm.frost.compiler import _TMA_STORE_EPI_PIPELINES
+
     tmpl = pathlib.Path(cudnn.__file__).parent / "gemm" / "frost" / "kernel_templates"
     formulas = set()
     for f in sorted(tmpl.glob("sm*.py")):
         got = [l.strip() for l in f.read_text().split("\n") if l.strip().startswith("epi_subtile_elems = ")]
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)